scripts/deps-clean

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+docker-compose down
```

Comment:
Don't see the value in the indirection for 1 liner commands. It just makes figuring out what is actually going less obvious